ember-cli/2017-07/july-27.md

```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-06-22
+
+## Attendees
+
+
+
+## Agenda
```

Comment:
I would also like to discuss using native `Symbol` for private methods/properties going forward.  Mostly as a mitigation to an addon knowingly reaching in and modifying underscored properties/methods, and us eventually being worried about breaking the use case that the addon solves...